r/tests/testthat/test-dplyr-funcs-string.R

```diff
@@ -1075,6 +1093,15 @@ test_that("str_starts, str_ends, startsWith, endsWith", {
     df
   )
 
+  compare_dplyr_binding(
+    .input %>%
+      transmute(a = str_starts(x, "b.*"),
+                b = str_starts(x, "b.*", negate = TRUE),
+                d = str_starts(x, fixed("b"))) %>%
+      collect(),
+    df
+  )
+
```

Comment:
Ah, nice! I see you're already used to `compare_dplyr_binding` here. Nice additions!